Nodejs/Product/Nodejs/NodejsTools.vsct

```diff
@@ -59,6 +59,10 @@
         <Parent guid="guidNodeToolsWseContextMenuCmdSet" id="idmNpm"/>
       </Group>
 
+      <Group guid="guidMigrateToJspsCmdSet" id="cmdidJspsMigrate">
```

Comment:
Item #&#x2060;1 [above](https://github.com/microsoft/nodejstools/pull/2456#issue-1626392894) -- move command to separate .vsct file so that it is easily removable when it's no longer needed